boards/arm/colibri_imx7d_m4/colibri_imx7d_m4_defconfig

```diff
@@ -0,0 +1,15 @@
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_IMX=y
+CONFIG_SOC_SERIES_IMX7_M4=y
+CONFIG_SOC_MCIMX7_M4=y
+CONFIG_BOARD_COLIBRI_IMX7D_M4=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_UART_NXP_IMX=y
```

Comment:
Remove this line, we can select the appropriate uart driver in the SoC defconfig. See arch/arm/soc/nxp_imx/rt/Kconfig.defconfig.mimxrt1052 for an example.